src/pages/home/report/ReportActionCompose/SuggestionEmoji.tsx

```diff
@@ -227,7 +227,7 @@ function SuggestionEmoji(
         <EmojiSuggestions
             highlightedEmojiIndex={highlightedEmojiIndex}
             emojis={suggestionValues.suggestedEmojis}
-            prefix={value.slice(suggestionValues.colonIndex + 1, selection.start)}
+            prefix={value.slice(suggestionValues.colonIndex + 1, selection.end)}
```

Comment:
Just left a comment in case someone is confused about this change. 

I think it makes sense to use `selection.end`, see below comparison 
<img width="532" alt="image" src="https://github.com/Expensify/App/assets/117511920/11328e52-3256-42c6-8341-cdb18eded661">

<img width="593" alt="image" src="https://github.com/Expensify/App/assets/117511920/3bd523be-e88e-403d-8d42-6f65393b2e0f">